UIKItTemplate/Modules/WebBrowser/Controller/WebViewController.swift

```diff
@@ -0,0 +1,157 @@
+        forwardButton.isEnabled = false
+        let refreshButton = UIBarButtonItem(systemItem: .refresh)
+        let flexibleSpace = UIBarButtonItem(systemItem: .flexibleSpace)
+        refreshButton.action = #selector(refreshAction)
+        return [backButton, forwardButton, flexibleSpace, refreshButton]
+    }
+
+    private func loadURL() {
+        guard let url = URL(string: urlString) else { return }
+        let urlRequest = URLRequest(url: url)
+        webView.load(urlRequest)
+    }
+
+    @objc private func closeViewController() {
+        dismiss(animated: true)
+    }
+
+    @objc private func backAction() {
+        if webView.canGoBack {
+            webView.goBack()
+        }
+    }
+
+    @objc private func forwardAction() {
+        if webView.canGoForward {
+            webView.goForward()
+        }
+    }
+
+    @objc private func refreshAction() {
+        webView.reload()
+    }
+}
+
+// MARK: - Constraints
+
+private extension WebViewController {
+    private func setupConstraints() {
+        setupWebViewConstraints()
+        setupWebToolBarConstraints()
+        setupCloseButtonConstraints()
+    }
+
+    private func setupWebViewConstraints() {
+        NSLayoutConstraint.activate([
+            webView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            webView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            webView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor)
+        ])
+    }
+
+    private func setupWebToolBarConstraints() {
+        NSLayoutConstraint.activate([
+            webToolBar.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            webToolBar.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            webToolBar.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            webToolBar.topAnchor.constraint(equalTo: webView.bottomAnchor),
+        ])
+    }
+
+    private func setupCloseButtonConstraints() {
+        NSLayoutConstraint.activate([
+            closeButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 17),
+            closeButton.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 28),
+            closeButton.heightAnchor.constraint(equalToConstant: 24),
+            closeButton.widthAnchor.constraint(equalTo: closeButton.heightAnchor),
+        ])
+    }
+}
+
+// MARK: - WebViewController: WKNavigationDelegate
+
+extension WebViewController: WKNavigationDelegate {
+    // Enable/disable navigation buttons
+    func webView(_ webView: WKWebView, didCommit navigation: WKNavigation!) {
+        if let itemsCount = webToolBar.items?.count, itemsCount > 0, let backButton = webToolBar.items?[0] {
+            backButton.isEnabled = webView.canGoBack
+        }
+        if let itemsCount = webToolBar.items?.count, itemsCount > 0, let forwardButton = webToolBar.items?[1] {
+            forwardButton.isEnabled = webView.canGoForward
+        }
```

Comment:
Тут какая-то бесполезная проверка на > 0, то что больше нуля, еще не значит, что айтемов 2 и у тебя не отвалится webToolBar.items[1]

если ты точно знаешь, что у тебя два айтема, то ифы можно сильно упростить
```suggestion
if let toolbarItems = webToolBar.items {
            let backButton = toolbarItems[0]
            let forwardButton = toolbarItems[1]
            backButton.isEnabled = webView.canGoBack
            forwardButton.isEnabled = webView.canGoForward
        }
```

Reply:
Второе условие поправил. Оставил как есть - так безопасней. Если кто поменяет кол-во items - приложение не упадёт